src/Client/AcsfClient.php

```diff
@@ -59,11 +59,14 @@ public function ping(): bool {
   /**
    * Returns all the sites from the acsf subscription.
    *
+   * @param int $limit
+   *   Query limit.
+   *
    * @return array
    *   All the existing site the user has permission to view.
    */
-  public function listSites(): array {
-    $response = $this->getJsonResponseBody($this->get('sites'), 'No sites found.');
+  public function listSites(int $limit = 1000): array {
+    $response = $this->getJsonResponseBody($this->get("sites?limit={$limit}"), 'No sites found.');
```

Comment:
We should be using the API properly here:
```suggestion
    $response = $this->getJsonResponseBody($this->get('sites', [
      RequestOptions::QUERY => ['limit' => $limit]
    ]), 'No sites found.');
```